### PR TITLE
Add Rust GraphQL example repo

### PR DESCRIPTION
**Pull request by ronanyeah**

### CODE REVIEW

**Comment by palfrey** [comment]
Various issues with this repository:
- No CI (not a requirement, but always concerns me when missing)
- Build failures. I get multiple errors of the form
```
error[E0596]: cannot borrow `self` as mutable, as it is not declared as mutable
    --> /home/palfrey/.cargo/registry/src/github.com-1ecc6299db9ec823/futures-lite-1.10.0/src/io.rs:2819:24
     |
2815 |         self: Pin<&mut Self>,
     |         ---- help: consider changing this to be mutable: `mut self`
...
2819 |         Pin::new(&mut *self.0.lock().unwrap()).poll_read(cx, buf)
     |                        ^^^^ cannot borrow as mutable
```
when trying `cargo test`
* The instructions are confusing (although might make more sense if I'd used Hasura before, but never heard of it before now). It says the the link to https://github.com/tarbh-engineering/journal will help, but I'm seeing nothing useful there.

**Comment by ronanyeah** [comment]
- I can add CI :+1: 
- `cargo test` is building fine for me, v1.46.0
- The journal repo is an Elm app utilising a Rust + Hasura backend, as demonstrated in the repo I'm trying to add

**Comment by ronanyeah** [comment]
Failures were probably due to `schema.json` in the setup instructions not being created. I added a placeholder so build should be fine for you now.

CI badge added.

File: README.md [hunk 1]
```diff
@@ -417,6 +417,7 @@ See also [A comparison of operating systems written in Rust](https://github.com/
 * [svenstaro/miniserve](https://github.com/svenstaro/miniserve) — A small, self-contained cross-platform CLI tool that allows you to just grab the binary and serve some file(s) via HTTP [<img src="https://github.com/svenstaro/miniserve/workflows/CI/badge.svg?branch=master">](https://github.com/svenstaro/miniserve/actions)
 * [TheWaWaR/simple-http-server](https://github.com/TheWaWaR/simple-http-server) — simple static http server
 * [wyhaya/see](https://github.com/wyhaya/see) — Static HTTP file server [![Build Status](https://api.travis-ci.org/wyhaya/see.svg?branch=master)](https://travis-ci.org/wyhaya/see)
+* [ronanyeah/rust-hasura](https://github.com/ronanyeah/rust-hasura) — A demonstration of how a Rust GraphQL server can be used as a remote schema with [Hasura](https://hasura.io/) ![Build status](https://github.com/ronanyeah/rust-hasura/workflows/Rust/badge.svg)
```
Comment (palfrey) [inline README.md:420]: Please add the branch name to the build badge image as per https://github.com/rust-unofficial/awesome-rust/blob/master/CONTRIBUTING.md

**Pull request merged**